packages/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -331,4 +332,65 @@ export class MultisignatureTransaction extends BaseTransaction {
 
 		return [];
 	}
+
+	public addMultisignature(
+		store: StateStore,
```

Comment:
for registration of multisignature, it should be `_: StateStore` and not used

Reply:
Changed